### DISCUSSION TITLE
ストック一覧ページのカテゴリーを使いやすくしてほしい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#65] [要望・アイデア・不具合]
[ストック一覧ページ](https://qiita.com/stock)においてサイドバーにカテゴリーが表示されていて絞り込めるようになっていますが、タグは登録数の多い順にソートされているのに対し、カテゴリーはそうなっていません（作成順になってる？）
タグと同じく、登録数の多い順に表示されると使いやすくなるかと思います。
また、カテゴリーの数が多い場合省略表示され何回も「もっと見る」を押さなければならないので、そちらも初期表示数を多くするなどするといいかもしれません。

**Comment by kskwtnk** [comment]
@munierujp 

ご意見ありがとうございます！

## カテゴリーの表示順について

仰るとおり、カテゴリーは作成順でソートされています
こちらについては登録数順・アップデート順・自由に並び替え可能など多くの意見をいただいており、具体的な仕様としては固まっていませんが改修したいと考えています！

具体的なIssueにした際にはまたアナウンスいたします！
他の方で「こういう挙動が望ましい」があれば、コメントやvoteなどお願いします 🙏 

## 「もっと見る」ついて

こちらについても仰るとおりでして、件数の変更はしたいと考えています！

Discussionを起点にIssueを作成して改修していく流れも作っていきたいので、こういったご意見をいただけて感謝です！

**Comment by wataru86** [comment]
@munierujp
> また、カテゴリーの数が多い場合省略表示され何回も「もっと見る」を押さなければならないので、そちらも初期表示数を多くするなどするといいかもしれません。

お待たせしました！
ストック一覧ページのサイドバーに表示されるカテゴリーの初期表示数を増やしました :rocket:
https://qiita.com/release-notes/2023-06-14

**Comment by kimkim0814** [comment]
@munierujp
大変お待たせしました。

ストックリスト(旧カテゴリー)の並び替えができるようになりました。
これにより、ご自身の好きな順番でリストを整理できるようになっています。  
![](https://qiita-image-store.s3.ap-northeast-1.amazonaws.com/0/88/a5b4b0fb-4700-45fa-9a55-edbfe4b520d4.gif)

その他にも便利な機能をアップデートしていますので、ぜひご活用いただければ幸いです✨
https://blog.qiita.com/release-stock-share/
